application.go

```diff
@@ -96,6 +103,12 @@ func ApplicationFromConfig(isClient bool, file string) (Application, error) {
 		if !foundTarget {
 			return errMsg("No targets specified")
 		}
+
+		for name, _ := range def.TargetGroups {
```

Comment:
We should also check that all targets mentioned in TargetGroups are defined in Targets.